OHHTTPStubs/OHHTTPStubs.xcodeproj/project.pbxproj

```diff
@@ -1387,6 +1391,7 @@
 		09110A6519805F4800D175E4 /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				CURRENT_PROJECT_VERSION = 6.1.1;
```

Comment:
If you want to integrate a version bump of the library, you should do it at the Project level, not for each Target.
If you use the "Levels" view in the Build Settings you'll see that the Project's version is still set to `6.1.0`.
We have the version at the Project level to share this information and change it only in one place.
To fix this, change the version in the `OHHTTPStubs` project and remove the override on each target.